tests/processors/base/helpers/test_helpers.py

```diff
@@ -130,22 +129,40 @@ class TestLocationNormalizer(object):
     @pytest.mark.parametrize("test_input,expected", [
         # Locations normalized by ISO-3166 name standards
         ("Brazil", "Brazil"),
-        ("United States of America","United States"),
+        ("United States of America", "United States"),
 
         # Locations normalized by ISO-3166 acronym standards
-        ("US","United States"),
-        ("UK","United Kingdom"),
+        ("US", "United States"),
+        ("UK", "United Kingdom of Great Britain and Northern Ireland"),
 
         # Locations normalized by Levenshtein distance (name)
-        ("Chnia","China"),
-        ("The Netherlands","Netherlands"),
-        ("thauland","Thailand"),
+        ("Chnia", "China"),
+        ("The Netherlands", "Netherlands"),
+        ("thauland", "Thailand"),
 
         # Locations not normalized (corner cases)
-        ("Asia(except Japan)","Asia(except Japan)"),
-        ("Outside","Outside"),
+        ("Asia(except Japan)", "Asia(except Japan)"),
+        ("Outside", "Outside"),
         (None, None)])
 
 
     def test_location_normalizer(self, test_input, expected):
         assert helpers.get_canonical_location_name(test_input) == expected
+
+class TestOrganisationNormalizer(object):
+    @pytest.mark.parametrize("test_input,expected", [
+        # Organisation normalized with a longer equivalent
+        ("Ghent University", "Ghent University Hospital"),
+
+        # Organisation normalized as it is (the longest available form)
+        ("Justus Liebig University of Giessen",
+         "Justus Liebig University of Giessen"),
+
+         # New organisation non-normalized
+        ("Federal University of Campina Grande",
+         "Federal University of Campina Grande"),
+    ])
+
+    @pytest.mark.usefixtures('organisation_cluster')
+    def test_organisation_normalizer(self, conn, test_input, expected):
+        assert helpers.get_canonical_organisation_name(conn, test_input) == expected
```

Comment:
I think this test is doing too much. First of all, the pattern of the `organisation_cluster` fixture is confusing, as it simply adds two fixtures to the DB without returning anything (which is probably the reason you've used `@pytest.mark.usefixtures` instead of adding it as a parameter). There's also the issue that we're hardcoding the data we expect to be created by the fixture here. I think a fixture is the wrong abstraction here, at least with the very simple fixture tools we have.

Instead of doing this, please remove the `organisation_cluster` fixture and create the data you need manually in the tests for the 3 cases you're testing. You could create a `_create_organisation_cluster(self, conn, canonical_name, variations)` to help keep the code DRY.